art_tree.go

```diff
@@ -3,10 +3,13 @@ package art
 
 import (
 	"bytes"
+	"fmt"
 	_ "math"
 	_ "os"
 )
 
+var _ = fmt.Println
```

Comment:
This doesn't appear to be used anywhere; is this variable and the import of `fmt` necessary?  If not, I'd like to remove it :)